src/map/ai/controllers/player_controller.cpp

```diff
@@ -51,16 +52,19 @@ bool CPlayerController::Cast(uint16 targid, SpellID spellid)
     {
         return CController::Cast(targid, spellid);
     }
+
     else
+
     {
         PChar->pushPacket(new CMessageBasicPacket(PChar, PChar, 0, 0, MSGBASIC_UNABLE_TO_CAST));
+
         return false;
     }
 }
 
 bool CPlayerController::Engage(uint16 targid)
 {
-    //#TODO: pet engage/disengage
```

Comment:
that # is actually there on purpose, its related to a visual studio feature.

Reply:
Ahh so all TODO should be #TODO:

Reply:
I usually prefer the space between `//` and anything following it for readability. I dunno if all todo should be marked so I will summon our canuck leader for that question.

@teschnei *should* the `#` be there for all, or just the ones already marked?